drivers/kscan/kscan_gt911.c

```diff
@@ -242,7 +242,8 @@ static int gt911_init(const struct device *dev)
 		LOG_ERR("Could not configure int GPIO pin");
 		return r;
 	}
-
+	/* Delay at least 10 ms after power on before we configure gt911 */
+	k_sleep(K_MSEC(20));
```

Comment:
why do we say "... delay at least 10 ms..." but then use 20msec instead of 10?

Reply:
I took this delay from our MCUXpresso SDK driver for the GT911, I assume that 20msec is used as a safe margin, since the GT911 datasheet requires at least 10msec of delay before any pins are configured after boot but does not impose a maximum delay